src/views/applicationManager.js

```diff
@@ -1,24 +1,56 @@
 const interview = {};
-interview.navigate = function(page) {
-    const appEl = document.querySelector('#app');
-    switch (page) {
-    case 'evaluations':
-        appEl.innerHTML = interview.EvaluationsPage();
-        interview.AddEventsNavigation();
-        //evaluationsEvents();
-        break;
 
-    case 'newEvaluation':
-        appEl.innerHTML = interview.newEvaluationPage();
-        interview.formEventsSetup();
-        interview.AddEventsNavigation();
-        break;
+// find page
+page = [];
+if (page == 0) {
+    page = 'login';
+} 
+else {
+//   destroy(lastPage);
+//    init(newPage);
+}
+//modules
+interview.modules = {
+    moduleEvaluations: {
+        init() {
+            const container = document.querySelector('#app');
+            //render
+            container.innerHTML = interview.EvaluationsPage();
+            //events
+            interview.AddEventsNavigation();
+        },
+    },
 
-    case 'login':
-        appEl.innerHTML = interview.LoginPage();
-        interview.AddEventsLogin();
-        break;
+    moduleNewEvaluation: {
+        init() {
+            const container = document.querySelector('#app');
+            //render
+            container.innerHTML = interview.newEvaluationPage();
+            //events
+            interview.formEventsSetup();
+            interview.AddEventsNavigation();
+        },
+    },
 
-    default: 
+    moduleLogin: {
+        init() {
+            const container = document.querySelector('#app');
+            //render
+            container.innerHTML = interview.LoginPage();
+            //events
+            interview.AddEventsLogin();
+        },
     }
-}
+
+    /*const removeEvents = {
```

Comment:
If we're not using it, it can probably go away, yes?

Reply:
I was trying to use it for destroy buuut I couldn't manage to make it work so I commented it for later.